src/imagetilesource.js

```diff
@@ -114,8 +114,9 @@
             }
 
             $.addEvent(image, 'load', function () {
-                _this.width = image.naturalWidth;
-                _this.height = image.naturalHeight;
+                /* IE8 fix since it has no naturalWidth and naturalHeight */
+                _this.width = Object.prototype.hasOwnProperty.call(image, 'naturalWidth') ? image.naturalWidth : image.width;
```

Comment:
Sorry I'm late to the party here! This seems like a good fix, I just have a question about the best way to do it... it seems to me that `_this.width = image.naturalWidth || image.width` would be sufficient, and more readable. The one edge case is if image.naturalWidth is actually 0, but in that case we have bigger problems (a 0 width image isn't going to be very interesting to display).

If that's not sufficient, how about `image.hasOwnProperty('naturalWidth')` instead of doing the `call`? Does that not work on ie8?